reports/live-1-domains/main_test.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+
+	"k8s.io/api/networking/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
+)
+
+func Test_live1DomainSearch(t *testing.T) {
+	type args struct {
+		domainSearch *v1beta1.IngressList
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    []map[string]string
+		wantErr bool
+	}{
+		{
+			name: "live1DomainSearch-Success",
+			args: args{
+				domainSearch: &v1beta1.IngressList{
+					Items: []v1beta1.Ingress{
+						{
+							ObjectMeta: metav1.ObjectMeta{
+								Namespace: "namespace-1",
+								Name:      "ingress-1",
+							},
+							Spec: v1beta1.IngressSpec{
+								TLS: []v1beta1.IngressTLS{
+									{
+										Hosts: []string{"example.live-1.cloud-platform.service.justice.gov.uk"},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			want: []map[string]string{
+				{
+					"hostname":  "example.live-1.cloud-platform.service.justice.gov.uk",
+					"namespace": "namespace-1",
+					"ingress":   "ingress-1",
+				},
+			},
+			wantErr: false,
```

Comment:
This is great. You have table to test the "good" path of this function. It's also handy to test the "bad" path, which will allow future devs to catch errors. The "bad" path in this instance would be a host name without "live-1" defined and the list returning empty.